src/python/ensembl/io/genomio/gff3/extract_annotation.py

```diff
@@ -230,18 +239,20 @@ def _transfer_description_up(self, child_feature: str) -> None:
                     parent["description"] = child_description
 
     @staticmethod
-    def product_is_informative(product: str, feat_id: Optional[str] = None) -> bool:
+    def product_is_informative(product: str, feat_ids: Optional[List[str]] = None) -> bool:
         """Returns True if the product name contains informative words, False otherwise.
 
         It is considered uninformative when the description contains words such as "hypothetical" or
-        or "putative". If a feature ID is provided, consider it uninformative as well (we do not want
+        or "putative". If a feature IDs are provided, consider it uninformative as well (we do not want
         descriptions to be just the ID).
 
         Args:
             product: A product name.
-            feat_id: Feature ID (optional).
+            feat_ids: List of feature ID.
 
         """
+        if feat_ids is None:
+            feat_ids = []
```

Comment:
```suggestion
```
Not needed since you are checking if `feat_ids` is a list with something in it below.